src/main/java/net/mcreator/util/StringUtils.java

```diff
@@ -74,6 +80,10 @@ public static String camelToSnake(String original) {
 		return underscoreReducer.matcher(String.join("_", namePartsSplitter.split(original))).replaceAll("");
 	}
 
+	public static String snakeToCamel(String original) {

```

Comment:
Where is this one used?

Reply:
I started working on block states support, and there it will be used to create enum property names (`TestProperty`) from their registry names (`test_property`);
>that's the idea, anyway.

But IMO it belongs to this pull since it's targeted specially to updating this file (unlike the block states feature), so I put it here.